internal/ingress/annotations/authreq/main.go

```diff
@@ -143,6 +147,8 @@ func (a authReq) Parse(ing *extensions.Ingress) (interface{}, error) {
 	// Optional Parameters
 	signIn, _ := parser.GetStringAnnotation("auth-signin", ing)
 
+	authSnippet, _ := parser.GetStringAnnotation("auth-snippet", ing)
```

Comment:
Maybe We can log a Warning(glog.Warning) that it will not be set if `err != nil`.

Reply:
Do the same for signIn above.